.github/workflows/format.yml

```diff
@@ -0,0 +1,20 @@
+name: Formatting
+
+on:
+  push:
+    branches: [ master ]

```

Comment:
Doesn't need to happen on push to master, only on PRs to master, as we're doing trunk-based development.

Reply:
Done